src/it/java/com/google/checkstyle/test/base/AbstractModuleTestSupport.java

```diff
@@ -199,8 +204,10 @@ protected static DefaultConfiguration createTreeWalkerConfig(Configuration confi
      * @return {@link DefaultConfiguration} for the given {@link Configuration} instance.
      */
     protected static DefaultConfiguration createRootConfig(Configuration config) {
-        final DefaultConfiguration dc = new DefaultConfiguration("root");
-        dc.addChild(config);
+        final DefaultConfiguration dc = new DefaultConfiguration(ROOT_MODULE_NAME);
+        if (config != null) {
```

Comment:
why check for null become required, you do not pass null at AbstractXpathRegressionTest

Reply:
Yeah, I can revert it, it was one of my tries to fix errors

Btw, I took it from `AbstractModuleTestSupport` located in `test` package, should I revert it?

Reply:
Done